components/ProductGrid/index.jsx

```diff
@@ -18,27 +18,48 @@ import Layout from './components/Layout';
  * @param {Object} props The component props.
  * @returns {JSX}
  */
-const ProductGrid = ({ handleGetProducts, products, totalProductCount }) => (
-  <InfiniteContainer
-    wrapper={Layout}
-    iterator={Iterator}
-    loader={handleGetProducts}
-    items={products}
-    loadingIndicator={<LoadingIndicator />}
-    totalItems={totalProductCount}
-    initialLimit={6}
-    limit={ITEMS_PER_LOAD}
-  />
-);
+const ProductGrid = ({ flags, infiniteLoad, handleGetProducts, products, totalProductCount }) => {
+  if (infiniteLoad) {
```

Comment:
Please turn that around and make it a guard clause. `if (!infiniteLoad) { ... }`